ddtrace/internal/sma.py

```diff
@@ -55,8 +46,7 @@ def set(self, count, total):
         :param total: The total quantity of the next bucket.
         :type total: :obj:`int`
         """
-        if count > total:
-            raise ValueError
+        count = min(count, total)
```

Comment:
Does this do the same thing as the `if` statement? If I have:
```
self.size = 2
self.counts = [1, 2, 3, 4]
self.totals = [1, 3, 5, 7]
```
Then I would need to call:
```
my_sma.set(5, 9) # correct
```
But this is checking for if I'm incorrect like this?
```
my_sma.set(5, 4) # incorrect
```
And do this (in essence)?
```
my_sma.set(4, 4)
```


(I think my understanding of this SMA implementation is not super clear honestly, so sorry if this is a stupid question 😅)

Reply:
Seems right to me. The `min` call in this change says "if count is larger than total, instead of raising an error, snap the count to the total".